docs/documentation_guidelines.md

```diff
@@ -173,14 +173,25 @@ All links in content should follow these guidelines:
 * **Start link text with keywords:** Frontload the link text with the most important words to help users scan the text.
 
 Use following examples to write links in your documentation:
-* To add a cross-reference to another documentation page, use file path from [DoxygenLayout.xml](./openvino-documentation/blob/master/docs/doxygen/DoxygenLayout.xml) (don't use direct links to docs.openvinotoolkit.org):
+
+* To add a cross-reference to another documentation page stored in current repository, use the file path relative to the file you are working in
 ```
-[OpenVINO Installation on Linux](./docs/install_guides/installing-openvino-linux.md)
+[Convert Your Caffe* Model](../MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md)
 ```
-Currently links to headings in other markdown files are not supported. To refer to a specific section, you may use the following example:  
+
+* To add a cross-reference to another documentation page from different repository, use `url` from the [docs/doxygen/ie_docs.xml](./docs/doxygen/ie_docs.xml) file
+> **NOTE**: If the documentation page isn't presented there, search *ie_docs.xml* in https://github.com/openvinotoolkit/ repositories. Do not use direct links from docs.openvinotoolkit.org.
```

Comment:
Please remove this line. Currently there is no the "ie_docs.xml" file in other repos.